yb-voyager/cmd/getCommand.go

```diff
@@ -21,12 +21,12 @@ import (
 	"github.com/spf13/cobra"
 )
 
-var liveMigrationCommand = &cobra.Command{
-	Use:   "live-migration",
-	Short: "This command will display the report of any live migration workflow",
-	Long:  ``,
+var getCommand = &cobra.Command{
+	Use:   "get",
+	Short: "Gets the report of the migration.",
```

Comment:
`Get migration reports` ? For now, we can specify "reports" at this level, but if we add any other subcommands under "get", we should remember to change this.

Reply:
Yes, correct